managing-os/pentesting.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+title: Penetration Testing
+permalink: /doc/pentesting/
+---
+
+**Legal notice:**
+
+The usage of penetration testing tools outside your own laboratory environment requires the permission of the organization you attack. Penetration testing without such a permission can have legal consequences.
+
+To avoid such legal conflicts please refer to the [Code of Ethics](https://www.eccouncil.org/Support/code-of-ethics).
```

Comment:
Should be something like "EC-Council Code of Ethics" in order to not imply that this is Qubes' code of ethics.